src/main/java/spoon/support/reflect/declaration/CtInterfaceImpl.java

```diff
@@ -51,6 +56,41 @@ public boolean isInterface() {
 		return Collections.unmodifiableList(l);
 	}
 
+	@Override
+	public <R extends CtCodeElement> R partiallyEvaluate() {
+		return null;
+	}
```

Comment:
wouldn't this be better as an exception? Is there any use case where partially evaluating a local interface will make sense?

Reply:
Good catch, seems like I missed that method. I'll implement it the same way `CtClassImpl` implements it.

Reply:
ahh, yup. I misunderstood how partial evaluation worked